src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. You're only adding a margin to the right, which is good for avoiding margin collapse, but ensure that this approach is consistent throughout your other elements.